### DISCUSSION TITLE
Tags validation getting wrong validation message

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9630] [Help]
**The form.social.0 field must not be greater than 5 characters.**

```
TagsInput::make('social')
->label('')
->placeholder('Enter your social names')
->nestedRecursiveRules([
    'min:1',
    'max:5',
])
->validationAttribute(__('social names')),
```

<img width="511" alt="Screenshot 2023-11-13 at 22 31 40" src="https://github.com/filamentphp/filament/assets/121255432/2907f83d-0b56-4735-909f-5118c5e28fdc">

https://filamentphp.com/docs/3.x/forms/fields/tags-input#tags-validation

**Comment by leandrocfe** [comment]
`->validationAttribute('Custom name')` ?

**Reply by devhoussam1998** [reply]
I already use `->validationAttribute('Custom name')` still getting wrong validation message

**Reply by GamendeBrian** [reply]
I am also stuck with this one. Tried some things like the answer above. Anyone else have an idea how to fix this?

**Reply by leandrocfe** [reply]
Could you open a new topic (Q&A) and share the code please?
